src/components/popup/style.ts

```diff
@@ -1,55 +1,65 @@
 import { PUITheme } from 'utils/types';
 import { styled } from 'linaria/react';
 import { breakpoints } from 'styles/breakpoints';
+import { PopupSizesType } from './types';
 
-export const StyledPopupTitle = styled.div`
-  margin-right: 16px;
-`;
+const sizeOptions = {
+  sm: '300px',
+  md: '460px',
+  lg: '798px',
+};
 
-export const StyledPopupHeaderTitleRow = styled.div`
+export const StyledPopupHeader = styled.div<{
+  theme: PUITheme;
+}>`
   display: flex;
   justify-content: space-between;
+  align-items: center;
+  padding: 16px 24px;
```

Comment:
let's imagine if you want to customize that component, you have to change that property for each element (header, body, footer) and only once if set it to parent + check those values in figma pls(16px for mobile and 24px for desktop)